src/Compilers/Core/Portable/SourceGeneration/Nodes/BatchNode.cs

```diff
@@ -13,6 +13,8 @@ namespace Microsoft.CodeAnalysis
 {
     internal sealed class BatchNode<TInput> : IIncrementalGeneratorNode<ImmutableArray<TInput>>
     {
+        private static readonly string? s_tableType = typeof(ImmutableArray<TInput>).FullName;
```

Comment:
>private static readonly string? s_tableType = typeof(ImmutableArray<TInput>).FullName; [](http://example.com/codeflow?start=8&length=86)

If it is almost never used. Would it be better to convert this to a calculated property with cache?